<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(170, 255, 255)

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid rgb(12, 38, 127); /* Viền màu đỏ dày 3px */
border-radius: 9px;    /* Bo tròn góc */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/19.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid rgb(12, 38, 127); /* Viền màu đỏ dày 3px */
border-radius: 9px;    /* Bo tròn góc */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/18.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>10</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#48006d;&quot;&gt;TÓM LƯỢC THÔNG TIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid rgb(12, 38, 127); /* Viền màu đỏ dày 3px */
border-radius: 9px;    /* Bo tròn góc */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/17.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid rgb(12, 38, 127); /* Viền màu đỏ dày 3px */
border-radius: 9px;    /* Bo tròn góc */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/15.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid rgb(12, 38, 127); /* Viền màu đỏ dày 3px */
border-radius: 9px;    /* Bo tròn góc */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Pictures/16.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>751</width>
      <height>192</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Tên Bệnh Nhân :</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Tuổi:</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Triệu chứng:</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Bệnh:</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Đề Xuất Thuốc</string>
     </property>
    </row>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>531</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_luu">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 87 italic 10pt &quot;Segoe UI Black&quot;;

background: rgb(85, 255, 255)</string>
       </property>
       <property name="text">
        <string>Lưu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_open_f">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 87 italic 10pt &quot;Segoe UI Black&quot;;

background: rgb(85, 255, 255)</string>
       </property>
       <property name="text">
        <string>Thoát</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
